src/debug/createdump/crashinfo.cpp

```diff
@@ -558,52 +554,6 @@ CrashInfo::GetELFInfo(uint64_t baseAddress)
         }
     }
 
-    // Skip the "interpreter" module i.e. /lib64/ld-linux-x86-64.so.2 or ld-2.19.so. The in-memory section headers are 
```

Comment:
@mikem8361 what is the debugging related consequence of this change? If I understand it correctly, we are not including the .eh_frame/.eh_frame_hdr unwind info anymore.